dev/ci/ci-flocq.sh

```diff
@@ -5,4 +5,10 @@ ci_dir="$(dirname "$0")"
 
 git_download flocq
 
-( cd "${CI_BUILD_DIR}/flocq" && ( if [ ! -x ./configure ]; then autoconf && ./configure; fi ) && ./remake "-j${NJOBS}" && ./remake install )
+( cd "${CI_BUILD_DIR}/flocq" && \
+  ( if [ ! -x ./configure COQEXTRAFLAGS="-compat 8.13" ]; then \
```

Comment:
```suggestion
  ( if [ ! -x ./configure ]; then \
```

Reply:
Also you can drop the `&&` and `\`, we have `set -e` in the ci-common sourced above. The CI scripts use `&&` because of cargo culting pretty much, it doesn't matter for most but once you start wanting newlines it's worth changing IMO.